### PR TITLE
Invisible score

### PR DESCRIPTION
**Pull request by just4fun**
This MR will resolve https://github.com/IFS49F/poker-server/issues/1 with https://github.com/IFS49F/poker-server/pull/3 .

Need more local tests for edge cases.

### CODE REVIEW

File: src/pages/Room/Room.js [hunk 1]
```diff
@@ -62,13 +69,10 @@ class Room extends Component {
   };
 
   handleShow = () => {
-    this.setState(prevState => ({
-      show: true
-    }));
-    this.socket.emit('show', true);
+    this.socket.emit('show');
```
Comment (hyjk2000) [inline src/pages/Room/Room.js]: Is there any particular reasons behind this change? I would like to keep the ability to set the value to either `true` or `false`, instead of only being able to set it to `true`.

File: src/pages/Room/Room.js [hunk 2]
```diff
@@ -25,16 +25,23 @@ class Room extends Component {
...
-        me,
+      this.setState(prevState => ({
+        /*
+        * For `Show` and `Clear` actions, we should also update `me` state
+        * to sync up with Redis data.
+        *
+        * For other actions, there is no need, since we don't want the scores
+        * will send to clients, so we reserve previous state for `me`.
+        */
+        me: shouldUpdateMe ? me : prevState.me,
```
Comment (just4fun) [inline src/pages/Room/Room.js]: Good idea!

For the code you attached, I also set score to `me`, b/c we will use it for `Summary` component.

Please review the new commits I pushed.

File: src/pages/Room/Room.js (hunk 1 above)
Comment (just4fun) [inline src/pages/Room/Room.js]: Because we have two actions, `Show` and `Clear`. There is no need to pass `true` for `Show` action.

If we will abstract the two events into one, we can use a boolean that time.

File: src/pages/Room/Room.js (hunk 2 above)
Comment (just4fun) [inline src/pages/Room/Room.js]: Back end code has been updated accordingly.
